libraries/core-react/src/components/Table/Cell/index.tsx

```diff
@@ -16,19 +16,27 @@ export type CellProps = {
   | TdHTMLAttributes<HTMLTableDataCellElement>
   | ThHTMLAttributes<HTMLTableHeaderCellElement>
 )
+export const Cell = forwardRef<
+  HTMLTableDataCellElement | HTMLTableHeaderCellElement,
+  CellProps
+>(function Cell({ className, ...rest }, ref) {
+  const props = {
+    ...rest,
+    className,
```

Comment:
This className should not need to be deconstructed... Suggest just passing props as before and set `ref` on components